issuegenerator/main_test.go

```diff
@@ -108,3 +108,42 @@ func TestIngestArtifacts(t *testing.T) {
 	require.Equal(t, expectedTestResults, rg.testSuites)
 
 }
+
+func TestTrimPath(t *testing.T) {
+
+	tests := []struct {
+		name           string
+		owner          string
+		repo           string
+		module         string
+		expectedModule string
```

Comment:
in Go we usually name the expected value as `want`; reference: https://go.dev/wiki/CodeReviewComments#useful-test-failures

```suggestion
		wantModule string
```

Reply:
Thanks for the links. I've historically followed the `testify` language of `expected`, but I can see the value in following best practices over a package we're dependent upon. Fixed.